recipes/aiidalab-widgets-base/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "aiidalab-widgets-base" %}
+{% set version = "2.3.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiidalab_widgets_base-{{ version }}.tar.gz
+  sha256: 4a6d3c7218925cf73d14e9872a9501100519181f616e88fcc246fc519a26cb57
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=42
+    - wheel
+    - pip
+  run:
+    - python >=3.9
+    - pycifrw >=4.4,<5.dev0
+    - aiida-core >=2.1,<3
+    - ansi2html >=1.6,<2.dev0
+    - ase >=3.18,<4.dev0,<3.23
+    - bokeh >=2.0,<3.dev0
+    - humanfriendly >=10.0,<11.dev0
+    - ipytree >=0.2,<1.dev0
+    - traitlets >=5.9,<6.dev0
+    - ipywidgets >=7.7,<8.dev0
+    - widgetsnbextension <3.6.3
+    - pymysql >=0.9,<1.dev0
+    - nglview >=3.0,<3.1dev0
```

Comment:
I needed to constrain nglview version, because nglview>=3.1 have outdated dependencies on conda-forge. I've opened a PR with a fix:

https://github.com/conda-forge/nglview-feedstock/pull/67